.github/workflows/promote-ga.yml

```diff
@@ -58,7 +58,7 @@ jobs:
     env:
       AMBASSADOR_RELENG_NO_GUI: "1"
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/3
```

Comment:
`checkout@v` got accidentally removed

Reply:
Good catch, I actually had fixed that locally but forgot to push it.